### DISCUSSION TITLE
Plotting paths from coordinates

### DISCUSSION THREAD

**Discussion by roaldarbol** [walkerke/mapgl#53] [General]
Hi! Really excited to be trying out mapgl! I’m wondering whether it’s possible to plot paths/lines from coordinates? Hoping to plot running routes as is currently possible with leaflet (addPolylines).

**Comment by walkerke** [comment]
Definitely!  If you have an sf LINESTRING, the equivalent function is `add_line_layer()`: https://walker-data.com/mapgl/articles/layers-overview.html#line-layers

**Reply by roaldarbol** [reply]
Awesome! I can’t quite figure out how I would input the longitudes and latitudes from the example - I guess they’re implicit for the roads in the example. How would you do that?

So equivalent to:
```r
leaflet(df) |>
   addTiles() |>
   addPolylines(lng = ~ lng, lat = ~ lat)
```

**Reply by roaldarbol** [reply]
@walkerke I'd still be really keen to learn how to do the above! 😊
